setup.py

```diff
@@ -1,53 +1,2 @@
-from setuptools import setup
-from distutils.core import Extension
-from distutils.command.build import build
-import os
 from subprocess import call
-import multiprocessing
-from glob import glob
-
-nxppy = Extension('nxppy',
-                    sources = ['Mifare.c', 'nxppy.c'],
-                    extra_compile_args=['-O1'],
-                    extra_link_args=['-lwiringPi']
-)
-
-class build_nxppy(build):
-    def run(self):
-        def compile():
-            call( './get_nxpRdLib.sh', shell=True )
```

Comment:
When i tried to make the setup.py file in this way the compilation of the nxppy.so file did not work because of the Cflags which are necessary to compile the file.I tried to pass over the Cflags in the extra_compile_args function but it did not work maybe you know a solution.